README.md

```diff
@@ -149,6 +149,24 @@ If the old parameter is no longer present (maybe because it has been renamed and
 removed already), no parameters are overwritten. You don't need to remove obsolete
 parameters from the rename map once they have been renamed.
 
+### Skipping file in production mode
+
+You can enable the parameter handler only in development mode (default behavior):
```

Comment:
this is *not* the default behavior

Reply:
You are right, I don't know why I wrote this...